frontend/packages/kubevirt-plugin/src/components/modals/hardware-devices/HardwareDevicesModal.tsx

```diff
@@ -163,6 +163,7 @@ export const HardwareDevicesModal = withHandlePromise(
                     : t('kubevirt-plugin~No Host devices found')
                 }
                 isUserForbidden={isUserForbidden}
```

Comment:
```suggestion
                disabled={isUserForbidden}
                // or
                // editable={!isUserForbidden}
```
if a user is forbidden we "disable"  input fields (e.g. delete and add) of show the editable version of the list

Reply:
+1